### ISSUE TITLE
Puppet fails installation for the first time

### ISSUE THREAD

**Issue by micsinyei** [wazuh/wazuh#20973]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 4.7 | Puppet | All-in-one | Puppet | Ubuntu |

I have tried to install wazuh via puppet and for the first run I am getting an error for the indexer and the dashboard:
`
Error: Could not update: Execution of '/usr/bin/apt-get -q -y -o DPkg::Options::=--force-confold --force-yes install wazuh-indexer=4.7.0-1' returned 100: Reading package lists...
Building dependency tree...
Reading state information...
W: --force-yes is deprecated, use one of the options starting with --allow instead.
E: Unable to locate package wazuh-indexer
Error: /Stage[main]/Wazuh::Indexer/Package[wazuh-indexer]/ensure: change from 'purged' to '4.7.0-1' failed: Could not update: Execution of '/usr/bin/apt-get -q -y -o DPkg::Options::=--force-confold --force-yes install wazuh-indexer=4.7.0-1' returned 100: Reading package lists...
Building dependency tree...
Reading state information...
W: --force-yes is deprecated, use one of the options starting with --allow instead.
E: Unable to locate package wazuh-indexer
`
First the problem was the wazuh is using legacy facts. I updated my puppet.conf but I did not solved the problem. After this, I checked the code and I found some information, like in the indexer:
`$manage_repos = false, # Change to true when manager is not present.`

 ` # JVM options`
 ` $jvm_options_memory = '1g',`
`) {`
 ` if $manage_repos {`
    `include wazuh::repo`
    `if $facts['os']['family'] == 'Debian' {`
      `Class['wazuh::repo'] -> Class['apt::update'] -> Package['wazuh-indexer']`
    `} else {`
      `Class['wazuh::repo'] -> Package['wazuh-indexer']`
  `  }`
  `}`
It feels like a dependency is missing from somewhere or the manager should also update the repositories of the indexer and dashboard here:
```
# Install wazuh-repository

  if $manage_repos {
    # TODO: Allow filtering of EPEL requirement
    class { 'wazuh::repo':}
    if $::osfamily == 'Debian' {
      Class['wazuh::repo'] -> Class['apt::update'] -> Package[$wazuh::params_manager::server_package]
    } else {
      Class['wazuh::repo'] -> Package[$wazuh::params_manager::server_package]
    }
  }
```
To summarize it, the installation should not have an error.
